content/doc/developer/views/symbols.adoc

```diff
@@ -0,0 +1,55 @@
+---
+title: Symbols
+layout: developerguide
+---
+
+pass:[<span class="version">Available since Jenkins 2.TODO.</span>]
+
+image::/images/developer/views/symbols.svg[Selection of Symbols]
+
+View the complete list of Symbols available to use in Jenkins on link:https://github.com/jenkinsci/jenkins/tree/master/war/src/main/resources/images/symbols[GitHub].
+
+[.lead]
+Jenkins Symbols are an extensive and consistent collection of icons that you can use on Jenkins or in your plugin.
+Symbols are scalable, support different weights and also adapt to the user's theme.
+
+---
+
+=== Using Symbols
+
+Using Symbols in your view is simple. Use the existing `icon` component and set the `src`
+value to the symbol you want, prefixed with `"symbol-"`.
+
+*Jelly example:*
+[source, xml]
+----
+<l:icon src="symbol-search" />
+----
+
+*Groovy example:*
+[source, groovy]
+----
+l.icon(src: 'symbol-search')
+----
+
+It's possible to add alt text and custom classes to Symbols, for example:
+
+*Jelly example:*
+[source, xml]
+----
+<l:icon src="symbol-search" alt="Search" class="custom-class" />
+----
+
+*Groovy example:*
+[source, groovy]
+----
+l.icon(src: 'symbol-search', alt: 'Search', class: 'custom-class')
+----
+

```

Comment:
Are tooltips supported as well?

Reply:
They're not, my initial thoughts behind tooltips is that they should be on the symbols container instead (e.g. a button), as the button will have a larger hitbox making it easier to read the tooltip.